syncdb.drush.inc

```diff
@@ -154,10 +154,9 @@ function _syncdb_manual_import($local_dump_path, $tables) {
       $invocations[] = array(
         'site' => '@self',
         'command' => 'sql-query',
-        'options' => array(
-          'file' => $local_dump_path . '/' . $table,
-          'database' => drush_get_option('database'),
-        ),
+        'options' => drush_redispatch_get_options() + ['strict' => 0] + array(
+            'file' => $local_dump_path . '/' . $table,
+          ),
```

Comment:
> Strict needs to be disabled because some options from importdb can get passed along to sql-query where they are invalid. Core Drush uses this technique for its internal redispatches.

Got it, that makes sense!